Cartfile.private

```diff
@@ -1,3 +1,3 @@
-github "Quick/Quick" ~> 1.3
-github "Quick/Nimble" ~> 7.1
-github "AliSoftware/OHHTTPStubs" ~> 6.1
+github "Quick/Quick" ~> 2.1
```

Comment:
Need new majors for Swift 5 compatibility, although these now drop Swift 3 and ObjC support in testing.